### PR TITLE
remove node-uuid

### PR DESCRIPTION
**Pull request by matikrk**
* write function generateRandomString

### CODE REVIEW

File: src/js/generateRandomString.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// @flow
+
+const generateRandomString = () => Math.random().toString(32).slice(2);
```
Comment (mpacholec) [inline src/js/generateRandomString.js]: maybe this one: 🙂 
```js
import {randomBytes} from 'crypto';

export const getRandomString= (length = 16, encoding='hex') => randomBytes(length).toString(encoding);
```

Comment (matikrk) [reply src/js/generateRandomString.js]: We need a simple one. No need other imports.

Comment (mpacholec) [reply src/js/generateRandomString.js]: `crypto` is part of nodeJS itself so why not to use it :) - but this is just a suggestion

Comment (aju) [reply src/js/generateRandomString.js]: @mpacholec but we use it in browser ?:P

Comment (mpacholec) [reply src/js/generateRandomString.js]: @aju https://developer.mozilla.org/en-US/docs/Web/API/Window/crypto

**Pull request merged**